app/assets/javascripts/application.js

```diff
@@ -12,6 +12,7 @@
 //= require theater
 //= require validate_file_input_fields
 //= require main
+//= require observation_form_map
```

Comment:
I'm completely open to that! I've still got quite a bit to learn about Rails (and MO's) conventions. I can quickly adjust to include directly from the template.

What I generally do is compile all my JS from component/template specific files into one global JS file (concatenated, minified, cached etc.). Then I apply a specific class to the body (controller/action or page name) that I need to add functionality to. That way I don't have to dig through HTML templates to find the relevant JS when unfamiliar!